eks-distro-base/Makefile

```diff
@@ -7,15 +7,27 @@ BASE_IMAGE?=public.ecr.aws/amazonlinux/amazonlinux:2
 IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
 IMAGE_NAME?=eks-distro/base
 # This tag is overwritten in the prow job to point to the commit hash
-IMAGE_TAG?=$(shell git rev-parse HEAD | cut -c 1-8)
+IMAGE_TAG?=$(shell date "+%F-%s")
 IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
+REPO_ROOT="$(shell cd "$(shell dirname "${BASH_SOURCE[0]}")" && pwd -P)"
 
 ifeq ($(DEVELOPMENT),true)
 	BASE_IMAGE=amazonlinux:2
 endif
 
+.PHONY: check-update
+check-update:
+	./check-update.sh
+	RETURN_VALUE=$(shell cat /tmp/status/return_value)
```

Comment:
Instead of writing to a file, can we just do `RETURN_VALUE=$(shell ./check-update.sh; echo $$?)` and use that as the conditional, or omit the echo if it is able to store the exit code properly?

https://stackoverflow.com/questions/7394290/how-to-check-return-value-from-the-shell-directive

Reply:
Also let's leave the check-update to just call the script, and do the return value checking as part of the update target. That way, you can also put check-update as a dependency of local-images instead of explicitly calling the script.

Reply:
Yeah sure, makes sense.